spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/FunctionTests.scala

```diff
@@ -210,46 +357,46 @@ class FunctionTests extends CAPSTestSuite with ScanGraphInit{
           |month: 10,
           |day: 1,
           |hour: 12,
-          |minute: 50}) AS time""".stripMargin).records.toMaps should equal(
+          |minute: 50}) AS time""".stripMargin).records.toMapsWithCollectedEntities should equal(
         Bag(
           CypherMap("time" -> java.sql.Timestamp.valueOf("2015-10-01 12:50:00.0"))
         )
       )
     }
 
     it("throws an error if values of higher significance are omitted") {
-      val e1 = the [IllegalArgumentException] thrownBy caps.cypher("RETURN localdatetime({ minute: 50 })").records.toMaps
-      e1.getMessage should(include("valid significance order") and include("minute"))
+      val e1 = the [IllegalArgumentException] thrownBy caps.cypher("RETURN localdatetime({year: 2011, minute: 50 })").records.toMapsWithCollectedEntities
+      e1.getMessage should (include("valid significance order") and include("minute"))
 
-      val e2 = the [IllegalArgumentException] thrownBy caps.cypher("RETURN localdatetime({ year: 2018, hour: 12, second: 14 })").records.toMaps
-      e2.getMessage should(include("valid significance order") and include("year, hour, second"))
+      val e2 = the [IllegalArgumentException] thrownBy caps.cypher("RETURN localdatetime({ year: 2018, hour: 12, second: 14 })").records.toMapsWithCollectedEntities
+      e2.getMessage should (include("valid significance order") and include("year, hour, second"))
     }
 
     it("throws an error if the localdatetime string is malformed") {
       val e1 = the [IllegalArgumentException] thrownBy(
-        caps.cypher("RETURN localdatetime('2018-10-10T12:10:30:15')").records.toMaps
+        caps.cypher("RETURN localdatetime('2018-10-10T12:10:30:15')").records.toMapsWithCollectedEntities
       )
       e1.getMessage.should(include("valid time construction string") and include("12:10:30:15"))
 
       val e2 = the [IllegalArgumentException] thrownBy(
-        caps.cypher("RETURN localdatetime('20181010T1210301')").records.toMaps
+        caps.cypher("RETURN localdatetime('20181010T1210301')").records.toMapsWithCollectedEntities
       )
       e2.getMessage should(include("valid time construction string") and include("1210301"))
 
       val e3 = the [IllegalArgumentException] thrownBy(
-        caps.cypher("RETURN localdatetime('20181010123123T12:00')").records.toMaps
+        caps.cypher("RETURN localdatetime('20181010123123T12:00')").records.toMapsWithCollectedEntities
       )
       e3.getMessage should(include("valid date construction string") and include("20181010123123"))
     }
 
     it("compares two datetimes") {
-      caps.cypher("RETURN localdatetime(\"2015-10-10T00:00:00\") < localdatetime(\"2015-10-12T00:00:00\") AS time").records.toMaps should equal(
+      caps.cypher("RETURN localdatetime(\"2015-10-10T00:00:00\") < localdatetime(\"2015-10-12T00:00:00\") AS time").records.toMapsWithCollectedEntities should equal(
```

Comment:
We could use single quotes (`'`) to avoid having to escape the double quote in these strings.
```suggestion
      caps.cypher("RETURN localdatetime('2015-10-10T00:00:00') < localdatetime('2015-10-12T00:00:00') AS time").records.toMapsWithCollectedEntities should equal(
```